envs/example/defaults.yml

```diff
@@ -322,15 +323,15 @@ keystone:
     - name: heat
       type: orchestration
       description: 'Heat Orchestration API'
-      public_url: https://{{ endpoints.main }}:8005/v1/%(tenant_id)s
-      internal_url: https://{{ endpoints.main }}:8005/v1/%(tenant_id)s
-      admin_url: https://{{ endpoints.main }}:8005/v1/%(tenant_id)s
+      public_url: https://{{ endpoints.main }}:8004/v1/%(tenant_id)s
```

Comment:
For port change, it will not break the existing ursula to create stack since the heat endpoints and the configurations have been changed too. Let's have this request pulled in first before #1144. After this has been pulled in, the #1144 will come back to do only cosmetic changes to clean up the port # to make sure it is consistent with other component.